ground/gcs/src/plugins/boards_aeroquad/aq32.cpp

```diff
@@ -184,8 +181,6 @@ Core::IBoardType::InputType AQ32::getInputType()
     switch(settings.RcvrPort) {
     case HwAQ32::RCVRPORT_PPM:
         return INPUT_TYPE_PPM;
-    case HwAQ32::RCVRPORT_PWM:
-        return INPUT_TYPE_PWM;
     default:
         break;
     }
```

Comment:
I think in here there's a call which asks whether an input type supported-- and it should say false for PWM.